### ISSUE TITLE
Wallaby not reporting tests

### ISSUE THREAD

**Issue by IPWright83** [wallabyjs/public#1280]
### Issue description or question
I've got an odd situation. All the tests disappear from the Wallaby app when un-commenting a stub test that barely does anything. I've got a bunch of failing tests, and that's fine - they're all reported correctly in the Wallaby app.

I've then written a couple of stub tests, everything runs fine with them listed like below:

```
 describe("events", () => {
        it("correctly fires lock", () => {
            expect(true).toBe(false);
        });

        // it("correctly fires unlock", () => {
        //     expect(true).toBe(false);
        // });
    });
```

If I un-comment the 2nd test however, then the Wallaby app no longer reports any tests and Wallaby itself within the IDE seems to struggle, despite the fact it seems to have logged the fact that it ran all 14 tests.

![image](https://user-images.githubusercontent.com/1374775/29863889-ef45376e-8d68-11e7-82bd-6ecf4649c09f.png)

```
​​​​​[Info]: Finished executing 13 affected test(s)​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:41:19 GMT wallaby:project Test run finished​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:41:19 GMT wallaby:project Processed console.log entries​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:41:19 GMT wallaby:project Processed loading sequences​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:41:19 GMT wallaby:project Processed executed tests​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:41:19 GMT wallaby:project Processed code coverage​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:41:19 GMT wallaby:project Test run result processed and sent to IDE​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:41:19 GMT wallaby:uiService Outgoing message ui:summary​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:41:19 GMT wallaby:uiService Outgoing message ui:tests:allResultsUpdated​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:41:19 GMT wallaby:uiService Outgoing message ui:coverageChanged​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:45 GMT wallaby:fs File changed in editor: packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:45 GMT wallaby:extended-core New document or complex document change​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:45 GMT wallaby:project File packages/galaxy/tests/specs/planet.unit.js changes are patches only: true​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:45 GMT wallaby:project Caching file [174] in memory​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:45 GMT wallaby:project Preparing to process packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:45 GMT wallaby:processor Running 1 processor(s) for packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:45 GMT wallaby:processor Finished running one processor for packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project Processing compiled packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project Instrumenting packages/galaxy/tests/specs/planet.unit.js, via process pool: false​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project No preprocessors configured for packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project Writing to disk and caching processed file packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:uiService Outgoing message ui:files​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:fs File changed: packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project File packages/galaxy/tests/specs/planet.unit.js changes saved, store ts: 1504082391872, change ts: 1504082745202​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project File packages/galaxy/tests/specs/planet.unit.js changes are patches only: true​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project Returning previously cached file [174] from memory​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:uiService Outgoing message ui:files​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project Preparing to process packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:processor Running 1 processor(s) for packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:processor Finished running one processor for packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project Processing compiled packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project Instrumenting packages/galaxy/tests/specs/planet.unit.js, via process pool: false​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project No preprocessors configured for packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project Writing to disk and caching processed file packages/galaxy/tests/specs/planet.unit.js​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project File packages/galaxy/tests/specs/planet.unit.js changes saved, store ts: 1504082391872, change ts: 1504082746218​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:project Test run started; run priority: 2​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:testTask Test files from affected: 1, from deleted or manually requested: 0, from recently changed: 0, from loaded by: 0, from failing: 0​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:workers Starting test run, priority: 2​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:nodeRunner Starting sandbox [worker #0, session #aieat]​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:nodeRunner Preparing sandbox [worker #0, session #aieat]​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:nodeRunner Prepared sandbox [worker #0, session #aieat]​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:46 GMT wallaby:workers Running tests in sandbox [worker #0, session #aieat]​​​​​
​​[Error]: Runtime error: TypeError: Cannot read property 'shadowImages' of undefined​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:47 GMT wallaby:workers Sandbox (active) [aieat] error: TypeError: Cannot read property 'shadowImages' of undefined​​​​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (.\packages\galaxy\src\d3\Planet.js:77:68)​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:47 GMT wallaby:workers Sandbox (active) [aieat] error: TypeError: Cannot read property 'shadowImages' of undefined​​​​​
​​[Error]:     at Selection.selection_each [as each] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:536:37)​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:47 GMT wallaby:workers Run 0 test(s), skipped 0 test(s)​​​​​
​​[Error]:     at Selection.selection_attr [as attr] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:593:15)​​
​​[Error]:     at C:\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:75:44​​
​​[Error]: Runtime error: TypeError: Cannot read property 'shadowImages' of undefined​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (.\packages\galaxy\src\d3\Planet.js:77:68)​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:577:19)​​
​​[Error]:     at Selection.selection_each [as each] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:536:37)​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:workers Sandbox (inactive) [aieat] error: TypeError: Cannot read property 'shadowImages' of undefined​​​​​
​​[Error]:     at Selection.selection_attr [as attr] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:593:15)​​
​​[Error]:     at C:\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:75:44TypeError: Cannot read property 'shadowImages' of undefined​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (C:\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:77:68)​​
​​[Error]:     at Selection.selection_attr [as attr] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:593:15)​​
​​​​​[Info]:     TypeError: Cannot read property 'shadowImages' of undefined​​​​​
​​[Error]:     at C:\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:75:44TypeError: Cannot read property 'shadowImages' of undefined​​
​​​​​[Info]:       ​​​​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (C:\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:77:68)​​
​​​​​[Info]:       at Planet.transitionShadow (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:205:65)​​​​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:577:19)​​
​​​​​[Info]:       at Transition.selection_call [as call] (node_modules\d3-selection\build\d3-selection.js:500:12)​​​​​
​​[Error]:     at Selection.selection_each [as each] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:536:37)​​
​​​​​[Info]:       at Planet.render (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:117:141)​​​​​
​​​​​[Info]:       at Object.<anonymous> (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\tests\specs\planet.unit.js:122:47)​​​​​
​​[Error]:     at Selection.selection_attr [as attr] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:593:15)​​
​​[Error]:     at C:\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:75:44​​
​​​​​[Info]:       at Object.e.length.arguments.(anonymous function) (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\wallaby\runners\node\jest@0.4.3\jasmine2Initializer.js:14:2006)​​​​​
​​​​​[Info]: ​​​​​
​​[Error]: Runtime error: TypeError: Cannot read property 'shadowImages' of undefined​​
​​​​​[Info]:   ● Planet › render › correctly renders a selected and locked planet​​​​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (.\packages\galaxy\src\d3\Planet.js:77:68)​​
​​​​​[Info]: ​​​​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:577:19)​​
​​​​​[Info]:       ​​​​​
​​[Error]:     at Selection.selection_attr [as attr] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:593:15)​​
​​​​​[Info]:       at Planet.transitionShadow (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:205:65)​​​​​
​​​​​[Info]:       at Transition.selection_call [as call] (node_modules\d3-selection\build\d3-selection.js:500:12)​​​​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (.\packages\galaxy\src\d3\Planet.js:77:68)​​
​​​​​[Info]:       at Object.<anonymous> (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\tests\specs\planet.unit.js:165:47)​​​​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:577:19)​​
​​​​​[Info]:       at Object.e.length.arguments.(anonymous function) (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\wallaby\runners\node\jest@0.4.3\jasmine2Initializer.js:14:2006)​​​​​
​​[Error]:     at Selection.selection_each [as each] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:536:37)​​
​​[Error]:     at Selection.selection_attr [as attr] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:593:15)​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:     expect(received).toBe(expected)​​​​​
​​​​​[Info]:     ​​​​​
​​​​​[Info]:     Expected value to be (using ===):​​​​​
​​​​​[Info]:       false​​​​​
​​​​​[Info]:     Received:​​​​​
​​​​​[Info]:       true​​​​​
​​​​​[Info]:       ​​​​​
​​​​​[Info]:       at Object.<anonymous> (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\tests\specs\planet.unit.js:170:41)​​​​​
​​​​​[Info]:       at Object.e.length.arguments.(anonymous function) (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\wallaby\runners\node\jest@0.4.3\jasmine2Initializer.js:14:2006)​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:   ● Planet › render › when undoing an action › correctly renders unselected planet​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:     expect(received).toBe(expected)​​​​​
​​​​​[Info]:     ​​​​​
​​​​​[Info]:     Expected value to be (using ===):​​​​​
​​​​​[Info]:       false​​​​​
​​​​​[Info]:     Received:​​​​​
​​​​​[Info]:       true​​​​​
​​​​​[Info]:       ​​​​​
​​​​​[Info]:       at Object.<anonymous> (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\tests\specs\planet.unit.js:176:45)​​​​​
​​​​​[Info]:       at Object.e.length.arguments.(anonymous function) (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\wallaby\runners\node\jest@0.4.3\jasmine2Initializer.js:14:2006)​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:   ● Planet › render › when undoing an action › correctly renders unlocked planet​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:     expect(received).toBe(expected)​​​​​
​​​​​[Info]:     ​​​​​
​​​​​[Info]:     Expected value to be (using ===):​​​​​
​​​​​[Info]:       false​​​​​
​​​​​[Info]:     Received:​​​​​
​​​​​[Info]:       true​​​​​
​​​​​[Info]:       ​​​​​
​​​​​[Info]:       at Object.<anonymous> (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\tests\specs\planet.unit.js:180:45)​​​​​
​​​​​[Info]:       at Object.e.length.arguments.(anonymous function) (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\wallaby\runners\node\jest@0.4.3\jasmine2Initializer.js:14:2006)​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:   ● Planet › render › when undoing an action › correctly renders unlocked and unselected planet​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:     expect(received).toBe(expected)​​​​​
​​​​​[Info]:     ​​​​​
​​​​​[Info]:     Expected value to be (using ===):​​​​​
​​​​​[Info]:       false​​​​​
​​​​​[Info]:     Received:​​​​​
​​​​​[Info]:       true​​​​​
​​​​​[Info]:       ​​​​​
​​​​​[Info]:       at Object.<anonymous> (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\tests\specs\planet.unit.js:184:45)​​​​​
​​​​​[Info]:       at Object.e.length.arguments.(anonymous function) (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\wallaby\runners\node\jest@0.4.3\jasmine2Initializer.js:14:2006)​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:   ● Planet › events › correctly fires lock​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:     expect(received).toBe(expected)​​​​​
​​​​​[Info]:     ​​​​​
​​​​​[Info]:     Expected value to be (using ===):​​​​​
​​​​​[Info]:       false​​​​​
​​​​​[Info]:     Received:​​​​​
​​​​​[Info]:       true​​​​​
​​​​​[Info]:       ​​​​​
​​​​​[Info]:       at Object.<anonymous> (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\tests\specs\planet.unit.js:192:41)​​​​​
​​​​​[Info]:       at Object.e.length.arguments.(anonymous function) (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\wallaby\runners\node\jest@0.4.3\jasmine2Initializer.js:14:2006)​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:   ● Planet › events › correctly fires unlock​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]:     expect(received).toBe(expected)​​​​​
​​​​​[Info]:     ​​​​​
​​​​​[Info]:     Expected value to be (using ===):​​​​​
​​​​​[Info]:       false​​​​​
​​​​​[Info]:     Received:​​​​​
​​​​​[Info]:       true​​​​​
​​​​​[Info]:       ​​​​​
​​​​​[Info]:       at Object.<anonymous> (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\tests\specs\planet.unit.js:196:42)​​​​​
​​​​​[Info]:       at Object.e.length.arguments.(anonymous function) (..\..\..\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\wallaby\runners\node\jest@0.4.3\jasmine2Initializer.js:14:2006)​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]: console.log: Test Suites: 1 failed, 1 total​​​​​
​​​​​[Info]: Tests:       8 failed, 6 passed, 14 total​​​​​
​​​​​[Info]: Snapshots:   3 passed, 3 total​​​​​
​​​​​[Info]: Time:        1.571s, estimated 7s​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]: ​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:workers Sandbox [aieat] is responsive, closing it​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:project Test run finished​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:project Processed console.log entries​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:project Processed loading sequences​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:project Processed executed tests​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:project Processed code coverage​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:project Test run result processed and sent to IDE​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:uiService Outgoing message ui:summary​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:uiService Outgoing message ui:tests:someResultsUpdated​​​​​
​​​​​[Info]: Wed, 30 Aug 2017 08:45:48 GMT wallaby:uiService Outgoing message ui:coverageChanged​​​​​
```

### Wallaby.js configuration file

```javascript
module.exports = function (wallaby) {
    return {
        name: "galaxy",
        files: [
            { pattern: "node_modules/jquery/dist/jquery.js", instrument: false },
            { pattern: "node_modules/babel-polyfill/dist/polyfill.js", instrument: false },
            { pattern: "packages/ui-core/src/**/*.js", load: false },
            { pattern: "packages/javascript-core/src/**/*.js", load: false },
            { pattern: "packages/context/src/**/*.js", load: false },
            { pattern: "packages/testHelpers/src/**/*.js", load: false },
            { pattern: "packages/galaxy/src/**/*.js", load: false, instrument: true },
            { pattern: "packages/galaxy/tests/data/*.json", load: false },
            { pattern: "packages/galaxy/tests/preProcessors/*.js", load: false },
            { pattern: "packages/galaxy/tests/mocks/*.js", load: false },
            { pattern: "packages/galaxy/tests/*.js", load: false },
            { pattern: "packages/galaxy/tests/**/*.snap", load: false, instrument: false },
            { pattern: "packages/galaxy/src/**/*.less", load: false, instrument: false },
        ],
        tests: [
            { pattern: "packages/galaxy/tests/specs/**/*.unit.js" },
        ],
        filesWithNoCoverageCalculated: [
            "!packages/galaxy/**/*.js",
        ],
        env: {
            type: "node",
            runner: "node",
        },
        testFramework: "jest",
        compilers: {
            "**/*.js": wallaby.compilers.babel({ babel: require("babel-core") }),
        },
        debug: true,
        setup: (target) => {
            target.testFramework.configure({
                moduleNameMapper: {
                    ".*.(less|css)$": require("path").join(target.localProjectDir, "/packages/galaxy/tests/preProcessors/jest.js"),
                },
            });
        },
    };
};

```

### Code editor or IDE name and version

Sublime Text v3 build 3142

### OS name and version

Windows 10 x64

**Comment by ArtemGovorov** [comment]
It looks like you have some global runtime error (occurring before tests execution), that causes the tests to be reported incorrectly. Here is the error:

```
[Error]: Runtime error: TypeError: Cannot read property 'shadowImages' of undefined​​
​​[Error]:     at HTMLUnknownElement.<anonymous> (.\packages\galaxy\src\d3\Planet.js:77:68)​​
​​​​​[Error]:     at Selection.selection_each [as each] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:536:37)​​
​​​​​[Error]:     at Selection.selection_attr [as attr] (C:\source\mood\edge-client\node_modules\d3-selection\build\d3-selection.js:593:15)​​
​​[Error]:     at C:\Users\ian.wright\AppData\Roaming\Sublime Text 3\Packages\Wallaby\projects\d2c7acd257329231\instrumented\packages\galaxy\src\d3\Planet.js:75:44​​
```

Is the `packages\galaxy\src\d3\Planet.js` imported by tests? Looks like the error is raised before some of your tests are executed.

**Comment by IPWright83** [comment]
Yeah `Planet.js` is imported - and I would expect that error regarding `shadowImages` but only as part of an actual test run. It shouldn't be possible for that code to be hit outside of an `it()`

**Comment by ArtemGovorov** [comment]
>  It shouldn't be possible for that code to be hit outside of an it()

From how the error is logged, it looks like it is somehow raised outside of a executing test context, maybe because it an async test (and Jest doesn't still properly support reporting for them). 

Just to make sure the root of the issue is in the error, could you please fix it, and then un-comment the 2nd test here:

```
 describe("events", () => {
        it("correctly fires lock", () => {
            expect(true).toBe(false);
        });

        // it("correctly fires unlock", () => {
        //     expect(true).toBe(false);
        // });
    });
```

to see if everything is reported correctly?

Could you please also paste the code of the test that may be causing the `shadowImages` error? I'll try to re-create a similar test to see if I can reproduce the issue.

**Comment by IPWright83** [comment]
Here is some sample code producing the error:

```
it("correctly renders selected planet", () => {
            const svg = createDOMElement("svg");
            const planet = new Planet();

            // Don't load shadows, and return custom data
            spyOn(planet, "loadShadowImages").and.returnValue(loadShadowsPromise);
            spyOnGetter(planet, "data").and.returnValue({
                id: "AA",
                type: "TEST",
                name: "Test Planet",
                isSelected: true,
            });

            planet.append(svg).render(true);
            expect(svg.innerHTML).toMatchSnapshot();
        });
```

And just to prove that the error should sit within an `it` here is a screenshot of the error being correctly reported.

A quick breakdown of what's going on:

1) `createDOMElement`  just creates a DOM element using ReactTestUtils
2) Constructor for a new Planet instance - nothing funny going on there
3) Using Jasmine's `spyOn`
4) Using a custom Spy - code included below
5) Adding some stuff to the DOM and calling render - this should fail synchronously
6) The exceptiations

here is the `spyOnGetter`

```
export function spyOnGetter(obj, prop) {
    const env = jasmine.getEnv();

    const spy = jasmine.createSpy(`${prop} spy`);
    Object.defineProperty(obj, prop, {
        get: spy,
        configurable: true,
    });

    // TODO - this doesn't actually clean up properly,
    // take a look at http://stackoverflow.com/questions/33148496/how-can-i-spy-on-a-getter-property-using-jasmine/38259654#38259654
    if (!handlerInstalled) {
        handlerInstalled = true;
        env.afterEach(() => afterHandler());
    }
    return spy;
}
```

**Comment by ArtemGovorov** [comment]
Thanks! 

A couple more questions:
- is the block of code
```
 describe("events", () => {
        it("correctly fires lock", () => {
            expect(true).toBe(false);
        });

        // it("correctly fires unlock", () => {
        //     expect(true).toBe(false);
        // });
    });
```
located in the same file as the failing test above?
- is there only one spec file, or more than one?

**Comment by IPWright83** [comment]
1) Yes the code is located in the same file
2) Yes, at the moment there is only 1 spec file being included.

**Comment by ArtemGovorov** [comment]
I have set up this repo: https://github.com/wallabyjs/wallaby-jest-sample/tree/1280 branch, however I can't reproduce the issue by commenting/uncommenting [this test](https://github.com/wallabyjs/wallaby-jest-sample/blob/1280/packages/galaxy/tests/specs/planet.unit.js#L28-L30). 

Could you please clone the branch to see if you can reproduce it?

I also have one more guess to check. In your wallaby config, try this setting:

```diff
  module.exports = function () {

    return {
      ...
+     reportUnhandledPromises: false
    };
  };
```
and run your scenario to see how it goes.

**Comment by IPWright83** [comment]
I've tried your branch, and unfortunately couldn't get a reproduction either. Even after copying my full test file (I noticed that it was very fragile as to whether or not I could reproduce it - commenting out other passing tests, and empty `it()` seemed to stop the issue).

Changing `reportUnhandledPromises` doesn't seem to fix it unfortunately. My codebase has moved on a bit, so I need to update my tests and I'll try again then.
